common/src/main/java/com/aliyun/emr/rss/common/network/server/MemoryTracker.java

```diff
@@ -45,23 +45,22 @@ public class MemoryTracker {
   private final long maxSortMemory;
   private final List<MemoryTrackerListener> memoryTrackerListeners = new ArrayList<>();
 
-  private final ScheduledExecutorService checkService = Executors
-    .newSingleThreadScheduledExecutor(new ThreadFactoryBuilder()
-      .setNameFormat("MemoryTracker-check-thread").build());
-  private final ScheduledExecutorService reportService = Executors
-    .newSingleThreadScheduledExecutor(new ThreadFactoryBuilder()
-      .setNameFormat("MemoryTracker-report-thread").build());
-  private final ExecutorService actionService = Executors
-    .newSingleThreadScheduledExecutor(new ThreadFactoryBuilder()
-      .setNameFormat("MemoryTracker-action-thread").build());
+  private final ScheduledExecutorService checkService =
+      ThreadUtils.newDaemonSingleThreadScheduledExecutor("MemoryTracker-check-thread");
+  private final ScheduledExecutorService reportService =
+      ThreadUtils.newDaemonSingleThreadScheduledExecutor("MemoryTracker-report-thread");
+  private final ExecutorService actionService =
+      ThreadUtils.newDaemonSingleThreadExecutorWithUncaughtExceptionHandler(
+          "MemoryTracker-action-thread", (t, e) ->
+              logger.error("memory-manager-actor thread {} has exception {}", t, e));
```

Comment:
it's better to use "memory-manager-action thread"  instead of "memory-manager-actor"?